execution/native/native_test.go

```diff
@@ -0,0 +1,14 @@
+package native
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestDefaultNatives(t *testing.T) {
+	ns, err := DefaultNatives()
+	require.NoError(t, err)
+	fmt.Println(ns)
```

Comment:
Can we test this?

Reply:
this is probably debug, I think all I really want to test is the invocation not `String()`.